src/main/java/twilightforest/client/model/armor/FieryArmorModel.java

```diff
@@ -12,7 +12,7 @@ public FieryArmorModel(ModelPart part) {
 
 	@Override
 	public void renderToBuffer(PoseStack stack, VertexConsumer builder, int light, int overlay, int color) {
-		super.renderToBuffer(stack, builder, 0xF000F0, overlay);
+		super.renderToBuffer(stack, builder, 0xF000F0, overlay, 654311423);
```

Comment:
Why 0x26FFFFFF ? Shouldn't it be 0xFFFFFFFF ?
The super uses -1 which is 0xFFFFFFFF.

Reply:
Also use hex for colors as its much more readable than decimal.